### ISSUE TITLE
Adding loan preparations doesn't work

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#2135]
https://user-images.githubusercontent.com/37256050/189771842-d2df1fb0-daa4-4dfa-bad1-6a864071f712.mov

Adding a loan preparation creates a read-only item that does not allow linking to an existing CO record. 

In Specify 6, this works properly by having the ability to enter items or add items from a record set:

<img width="1146" alt="image" src="https://user-images.githubusercontent.com/37256050/189772428-94ba3aca-4ef4-4e8b-bf3d-13e5d918eea6.png">

`v7.7.2`
`KUBirds`

**Comment by grantfitzsimmons** [comment]
@maxxxxxdlp This is working for Theresa when the form is being displayed in grid view

**Comment by maxxxxxdlp** [comment]
Yeah, there is special behavior of the grid view. Looks like we forgot to enable the special behavior for the form view

**Comment by CarolineDenis** [comment]
this is fixed on xml-editor
